### ISSUE TITLE
About the synthetic images

### ISSUE THREAD

**Issue by amoxicillinandmedlar** [AZZMM/CC-Diff#8]
Hello, thank you for your great work! I have some questions:
Are the synthetic images and checkpoints you provided for CC-Diff or CC-Diff++?
In infer_dior.py, when inferring on the validation set, you used the training set as background images. When inferring on the training set, do you use the validation set as background images?
